### PR TITLE
Added move to

### PR DESCRIPTION
**Pull request by DXXL**
| Q                | A
| ---------------- | ---
| Bug fix?         | no
| New feature?     | yes
| New tests added? | not needed
| Fixed tickets    | TODO
| License          | MIT

### Description

Adds a move to dialog which can move the current file and rename it.

### CODE REVIEW

File: src/main/actions/file.js [hunk 1]
```diff
@@ -134,6 +134,44 @@ ipcMain.on('AGANI::window::drop', (e, fileList) => {
   }
 })
 
+ipcMain.on('AGANI::response-file-move-to', (e, { pathname }) => {
+  const win = BrowserWindow.fromWebContents(e.sender)
+  if (pathname !== '') {
+    let newPath = dialog.showSaveDialog(win, {
+      buttonLabel: 'Move or rename',
+      nameFieldLabel: 'Filename:',
+      defaultPath: pathname || '~/Untitled.md'
```
Comment (Jocs) [inline src/main/actions/file.js]: The default path maybe the same as the `pathname`, so the `~/Untitled.md` is unnecessary?

```javascript
defaultPath: pathname
```

is OK ?

File: src/main/actions/file.js [hunk 2]
```diff
@@ -134,6 +134,44 @@ ipcMain.on('AGANI::window::drop', (e, fileList) => {
...
+    })
+    if (newPath === undefined) return
+    if (!fs.existsSync(newPath)) {
```
Comment (Jocs) [inline src/main/actions/file.js]: There is already a help function `isMarkdownFile`, maybe you can use it?

File: src/main/menus/file.js [hunk 3]
```diff
@@ -1,9 +1,9 @@
-import { app } from 'electron'
```
Comment (Jocs) [inline src/main/menus/file.js]: Aha, you maybe use JS format, But there should leave one space between `{` and `app`.

**Comment by DXXL** [comment]
Unfortunately, I can't use isMarkdownFile because the file is not yet created and therefore it is always returned false, but I need to check if the dialog was canceled or not. You are right, the style issue was caused by my IDE which messed up project and global settings. You are right if I check pathname beforehand I don't need to fall back to "Untitled" - fixed that.

**Comment by Jocs** [comment]
@DXXL Thanks，merged！

**Pull request merged**